src/ray/util/logging.h

```diff
@@ -101,7 +101,7 @@ enum class RayLogLevel {
 
 #define RAY_LOG_INTERNAL(level) ::ray::RayLog(__FILE__, __LINE__, level)
 
-#define RAY_LOG_ENABLED(level) ray::RayLog::IsLevelEnabled(ray::RayLogLevel::level)
+#define RAY_LOG_ENABLED(level) ::ray::RayLog::IsLevelEnabled(ray::RayLogLevel::level)
```

Comment:
Why prepend `::` ?

Reply:
Just to match the style of the rest of the file. It's not strictly needed for correct functionality.